HRIS.Services/Services/EmployeeService.cs

```diff
@@ -68,14 +68,7 @@ public async Task<EmployeeDto> CreateEmployee(EmployeeDto employeeDto)
 
         await _db.EmployeeRole.Add(new EmployeeRole(employeeRoleDto)); 
         
-        try
-        {
-            await _emailService.Send(new MailAddress(employeeDto.Email, $"{employeeDto.Name} {employeeDto.Surname}"), "WelcomeLetter", employeeDto);
-        }
-        catch (Exception ex)
-        {
-            _errorLoggingService.LogException(ex);
-        }
```

Comment:
Working? this was left in because the email service was broken.

Reply:
I put it back, sorry about that